images/docs-builder/links_whitelist.json

```diff
@@ -0,0 +1,9 @@
+{
+    "ignorePatterns": [
+        { "pattern": "^https://api.crc.testing" },
+        { "pattern": "^http://api.crc.testing" },
+        { "pattern": "https://console-openshift-console.apps-crc.testing" },
+        { "pattern": "http://example.proxy.com" }
```

Comment:
Totally unrelated to this PR, but `proxy.example.com` would be better than `example.proxy.com` as `example.com` is explicitly reserved for use in examples https://tools.ietf.org/html/rfc2606

Reply:
Simple enough change, added a commit to take care of this.